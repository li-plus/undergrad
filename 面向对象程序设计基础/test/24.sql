create database oop; 
use oop;
create table t(a int not null, b char);
insert into t(a, b) values (1, "x");
insert into t(a, b) values (1, "x");
insert into t(a, b) values (2, "x");
insert into t(a, b) values (3, "y");
insert into t(a, b) values (3, "z");
insert into t(a, b) values (3, "z");
select b, count(*) from t group by b order by count(*);
/********************** Sample Output *************************
b       COUNT(*)
y       1
z       2
x       3
**************************************************************/
select a, b, count(*) from t group by a, b order by count(*); 
/********************** Sample Output *************************
a       b       COUNT(*)
2       x       1
3       y       1
1       x       2
3       z       2
**************************************************************/
drop database oop;