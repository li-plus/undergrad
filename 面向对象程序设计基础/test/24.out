b       COUNT(*)
y       1
z       2
x       3
a       b       COUNT(*)
2       x       1
3       y       1
1       x       2
3       z       2
